bin/publish

```diff
@@ -124,6 +124,22 @@ rm -rf ui/*/.next
 pnpm install --prefer-frozen-lockfile
 git checkout pnpm-lock.yaml # the other github action will take care of this, but we need a clean working directory to publish
 
+# Check to make sure it was built correctly
+rm -rf core/dist
```

Comment:
```suggestion
```

was this left in from a test? looks like this explicitly removes the `dist` dir

Reply:
Nice catch, it was!